### ISSUE TITLE
Nemo can't change the folder icons

### ISSUE THREAD

**Issue by Palace4Software** [linuxmint/nemo#3196]
```
 * Nemo version 5.6.3
 * issue is with desktop AND windowed nemo
 * Distribution - Linux Mint 21.1
 * OpenSource Drivers (nouveau)
 * 64 bit
 ```

**Issue**
If I change the icon of a folder (not the color of the folder!) and then want to switch back to the old icon, the color selection no longer works.

**Steps to reproduce**
1. Start Nemo
2. Right-Click on a folder and choose the option "Properties"
3. Select in the new 'window' the icon and change it to something new
4. close the 'Properties' -window and reopen it
5. Select the icon of the folder and search in the icon-finder for 'folder'
6. Select the first entry and save the peferences
7. If you klick right now, you can see the buttons, to change the icons, but nothing happend, if you click them

**Expected behaviour**
If you klick right now, you can see the buttons, to change the icons, but nothing happend, if you click them.

**Other information**
- If you choose in step 6 the "stock_folder" icon, the issue will also still there.
- This issue is also in LMDE (Linux Mint Debian Edition) 5

**Comment by jousi76** [comment]
This happend to me also. 
```
* Nemo version 5.8.5
* Linux Mint 21.2 Cinnamon

```

**Comment by Jeremy7701** [comment]
In LMDE6:-

I did 1-6, Randomly changing a standard icon to "remote FTP" icon
Tried changing the colour - this works (and I get the old standard folder icon back)
Tried the remove colour option - and get the "remote FTP" icon back.
So I changed the colour to Green - and it looks just like my standard folder icons theme.

#7 Works, no problem, also if you want to change it back to the stock-folder, this also works.
**In short, can't reproduce the problem.**

So I suspect that, since colour only works for folder icons, that it has its own set of icons - possibly?
Have you tried updating to Mint 21.2 - possibly in a VM to check out the problem?
Maybe even try out LMDE6!!  ;-)

**Comment by Palace4Software** [comment]
Oh... I forgot I had this issue here.
I don't use cinnamon or Linux Mint [Debian Edition] anymore.
So if you're sure, that everything will work, I can close the issue.

**Comment by Jeremy7701** [comment]
It works as described for me - can't say about Mint 21.2, though.

**Comment by Palace4Software** [comment]
I think it should be the same nemo version (at least sometime). So I can close this.
